op-challenger/config/config.go

```diff
@@ -35,16 +35,30 @@ var (
 	ErrCannonNetworkUnknown          = errors.New("unknown cannon network")
 	ErrMissingRollupRpc              = errors.New("missing rollup rpc url")
 )
+var (
+	ErrMissingAsteriscL2               = errors.New("missing asterisc L2")
+	ErrMissingAsteriscBin              = errors.New("missing asterisc bin")
+	ErrMissingAsteriscServer           = errors.New("missing asterisc server")
+	ErrMissingAsteriscAbsolutePreState = errors.New("missing asterisc absolute pre-state")
+	ErrMissingAsteriscSnapshotFreq     = errors.New("missing asterisc snapshot freq")
+	ErrMissingAsteriscInfoFreq         = errors.New("missing asterisc info freq")
+	ErrMissingAsteriscRollupConfig     = errors.New("missing asterisc network or rollup config path")
+	ErrMissingAsteriscL2Genesis        = errors.New("missing asterisc network or l2 genesis path")
+	ErrAsteriscNetworkAndRollupConfig  = errors.New("only specify one of network or rollup config path")
+	ErrAsteriscNetworkAndL2Genesis     = errors.New("only specify one of network or l2 genesis path")
+	ErrAsteriscNetworkUnknown          = errors.New("unknown asterisc network")
+)
```

Comment:
Let's combine these variable blocks, but we can divide them with an empty line for readability.

```suggestion

	ErrMissingAsteriscL2               = errors.New("missing asterisc L2")
	ErrMissingAsteriscBin              = errors.New("missing asterisc bin")
	ErrMissingAsteriscServer           = errors.New("missing asterisc server")
	ErrMissingAsteriscAbsolutePreState = errors.New("missing asterisc absolute pre-state")
	ErrMissingAsteriscSnapshotFreq     = errors.New("missing asterisc snapshot freq")
	ErrMissingAsteriscInfoFreq         = errors.New("missing asterisc info freq")
	ErrMissingAsteriscRollupConfig     = errors.New("missing asterisc network or rollup config path")
	ErrMissingAsteriscL2Genesis        = errors.New("missing asterisc network or l2 genesis path")
	ErrAsteriscNetworkAndRollupConfig  = errors.New("only specify one of network or rollup config path")
	ErrAsteriscNetworkAndL2Genesis     = errors.New("only specify one of network or l2 genesis path")
	ErrAsteriscNetworkUnknown          = errors.New("unknown asterisc network")
)
```

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/10094/commits/ceb1b24643cb3cc9c50f7fc1c9b90f15096a729c.